Dockerfile.dash

```diff
@@ -2,6 +2,9 @@ FROM python:3.7.7-slim-buster
 
 WORKDIR /code 
 
+ARG PORT
+ENV PORT $PORT
```

Comment:
i might change this if it breaks the heroku build, but other than that i'll trust it